src/index.html

```diff
@@ -461,14 +485,21 @@ <h1 class="clear">Map boundary change</h1>
           </div>
         </div>
       </div>
+      <div title="Download map" class="map-download">
```

Comment:
can you remove all other changes and only include the map download button? 
that would be great. 
thanks. 
the other changes should be a separate PR.

Reply:
done